client/js/screens/LogIn/LogInContainer.js

```diff
@@ -22,11 +22,13 @@ class LoginContainer extends Component {
       <Mutation mutation={LoginMutation}>
         {login => (
           <UserContext.Consumer>
-            {({ signin, token }) => {
-              if (token) {
+            {({ storeSessionToken, token }) => {
+              if (token.length > 0) {
```

Comment:
if you make your Context state null initially you will not have to check for your token's length. You can just do `if(token)`